index.bs

```diff
@@ -4445,6 +4493,144 @@ Of the extended attributes defined in this specification, only the [{{Exposed}}]
 </div>
 
 
+<h3 id="idl-modules">Modules</h3>
+
+A <dfn id="dfn-module" export>module</dfn> is a structure that exposes a
+collection of interfaces, readonly attributes, and operations as a logical
+unit. A [=module=] has a <dfn for=module>specifier</dfn> that uniquely
+identifies it.
+
+For every string in an [=IDL fragment=] matching the <emu-nt><a
+href="#prod-Module">Module</a></emu-nt> nonterminal symbol, there must exist
+a [=module=] whose [=module/specifier=] is given by the <a
+href="#prod-string">string</a> immediately following the <emu-t>module</emu-t>
+terminal symbol.
+
+A module is a specification of a set of
+<dfn id="dfn-module-member" export lt="module member">module members</dfn>
+(matching <emu-nt><a href="#prod-ModuleMembers">ModuleMembers</a></emu-nt>),
+which are the [=interfaces=], [=regular operations=] and [=read only=] [=regular
+attributes=] that appear between the braces in the module declaration. These
+operations and attributes describe the behaviors packaged into the module.
+
+The IDL for modules can be split into multiple parts by using <dfn
+id="dfn-partial-module" export>partial module</dfn> definitions (matching
+<emu-t>partial</emu-t> <emu-nt><a href="#prod-Module">Module</a></emu-nt>).
+The module specifier of a partial module definition must be the same as the
+identifier of a module definition. All of the members that appear on each of
+the partial module definitions are considered to be members of the module
+itself.
+
+<pre class="syntax">
+    module <mark>"std:example"</mark> {
```

Comment:
As discussed previously, I am hopeful we can centralize the prefix in one place. (Whether it be HTML, or more likely it seems, Web IDL.) Having specs need to remember seems bad.

Reply:
There's a lot of open questions about how the module prefix will be determined, and at what level of specification, including:
https://github.com/tc39/proposal-javascript-standard-library/issues/14
https://github.com/tc39/proposal-javascript-standard-library/issues/12
https://github.com/tc39/proposal-javascript-standard-library/issues/20

I don't understand the "remember" point--presumably spec authors will have to use the module prefix in their sample code and tests. It doesn't seem like a lot to remember, either.

I'd like to separate the layering discussions from this PR, so I'll change these samples to avoid any particular prefix, and point to open issues, and suggest that we may do some kind of appending in the future, depending what's decided.

Is there any issue I can point to on the HTML/WebIDL side that includes your centralization idea?